### PR TITLE
Remove duplicate hero image on posts, update largo_byline on older templates

### PR DESCRIPTION
**Pull request by benlk**
## Changes

This pull request makes the following changes:

- Comments out `largo_hero()` call in the `single-two-column.php` template added for #99
- Updates the existing `partials/content-single-classic.php` and `partials/content-single.php` added in https://github.com/INN/umbrella-citylimits/pull/5 for compatibility with Largo 0.6.4 by adding the post ID to `largo_byline()` calls, in accordance with https://github.com/INN/largo/issues/1517
- Adds the post ID to all other `largo_byline()` calls in this theme
- Correct capitalization on all `get_post_id()` calls to `get_post_ID()`, to match the function declaration https://developer.wordpress.org/reference/functions/get_the_ID/

## Why

For #124

## Testing/Questions

Features that this PR affects:

- Article content

Questions that need to be answered before merging:

- [x] Is this PR targeting the correct branch in this repository?
- [ ] does it work?

Steps to test this PR:

1. Check it out.

### CODE REVIEW

**Comment by joshdarby** [comment]
@benlk I'm confused (probably because I don't have any background knowledge on the way they've been using featured media like you do), but wasn't the idea to use a built in Largo filter that would remove the image in the post if it matched the featured image?

**Comment by benlk** [comment]
The filter is presently removed from effect in https://github.com/INN/umbrella-citylimits/blob/1c2dbaaa13e4eb6cb17f89e64f9c5da1f3ef3617/wp-content/themes/citylimits/inc/post-templates.php#L20-L23, but I'm still seeing the media credits output upon the page. I'm going to download the relevant assets to see if the reason the filter is failing is because the image tags aren't being output due to lack of image.

**Comment by benlk** [comment]
tl;dr: the filter doesn't work on Gutenberg content. I'm writing up the rest of the problem in an issue in Largo. It boils down to the assumptions we made about those images' markup no longer being correct, in a https://stackoverflow.com/a/1732454 sort of way.

**Comment by benlk** [comment]
@joshdarby With https://github.com/INN/largo/issues/1834 being the case, this PR's removal of `largo_hero()`:

- works
- is maintainable

**Comment by joshdarby** [comment]
@benlk I'm fine with it, but we may want to write out a more in-depth tldr for @MirandaEcho and @kaylima so there's no confusion about images moving forward.

**Comment by benlk** [comment]
I don't think there's actually going to be any change for City Limits going forward; this is just making sure that the new article template continues the behavior of the past article templates.
